packages/core/src/utils/bigint.ts

```diff
@@ -75,7 +75,7 @@ export const bigIntClamp = (
  * @returns the base multiplied by the percent and divided by 100
  */
 export const bigIntPercent = (base: BigNumberish, percent: bigint) => {
```

Comment:
nit: we may want to make it clear in the function name or the docs that this is doing the rounding up

Reply:
yea I was gonna follow up with a PR that introduces an optional field called `RoundingMode` and default to up. blegh I'll do it in this one and include docs changes too